flutter/lib/src/default_integrations.dart

```diff
@@ -124,14 +124,14 @@ class FlutterErrorIntegration extends Integration<SentryFlutterOptions> {
   }
 
   @override
-  FutureOr<void> close() {
+  FutureOr<void> close() async {
     /// Restore default if the integration error is still set.
     if (FlutterError.onError == _integrationOnError) {
       FlutterError.onError = _defaultOnError;
       _defaultOnError = null;
       _integrationOnError = null;
     }
-    super.close();
+    await super.close();
```

Comment:
Instead of awaiting, you could remove the await and just `return super.close();`

Reply:
fair point